gulpfile.js

```diff
@@ -70,6 +70,32 @@ function handleError() {
   });
 }
 
+function createIndexFileStream() {
+  var stream = new PassThrough({ objectMode: true });
```

Comment:
This returns an `IndexFileStream` of all the `index.html` files in the site. It does a bit of trickery because we first need to generate the webpack bundle that *generates* the index files.

<img src="http://media.giphy.com/media/c3Xv5mtcTWde8/giphy.gif">